index.html

```diff
@@ -1,54 +1,121 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8">
-    <title>Fern's Resume</title>
-  </head>
-  <body>
-
-    <h2>Fern Hassan</h2>
-    <h3>
-      Graphic Designer and Illistrator 
-    </h3>
-    <h4>
-      Email: fern.hassan@gmail.com
-    </h4>
-    <h4>
-      Phone: 301-747-8766
-    </h4>
-    <h4>
-      <a href=https://www.linkedin.com/feed/> LinkedIn</a> 
-    </h4>
-  
-  
-
-    <p>Education</p>
-    <ul>
-      <li>Maryland Institute College of Art 
-Graphic Design BFA in Progress
-Expected June 2021
-</li>
-      <li>Montgomery College
-Graphic Design AFA, May 2019
-</li>
-    </ul>
-  
-  
-  <p>
-    Experience
-  </p>
-  
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="styles.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,900;1,400&display=swap" rel="stylesheet">
+  <title>Fern's Resume</title>
+</head>
+
+<body>
+  <h2>Fern Hassan</h2>
```

Comment:
Your code does not include an `h1`; see the HTML best practices re: [heading elements](https://mica-web.github.io/learn/html/best-practices#heading-tags-are-used-semantically-not-simply-for-presentation).